### PR TITLE
Tests for ShowNextCommand, chronological sort

### PR DESCRIPTION
**Pull request by lhy-hoyin**
- Tests for `ShowNextCommand`, `ShowNextCommandParser`
- Test that events are sorted during adding
- Test that events remain sorted when event is edited

### CODE REVIEW

File: src/test/java/ezschedule/logic/parser/ShowNextCommandParserTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        assertParseSuccess(parser, "   ", expected);
+    }
+
+    @Test
+    public void parse_validArgs_returnsShowNextCommandArbitraryUpcoming() {
+        assertParseSuccess(parser, "5", new ShowNextCommand(new UpcomingEventPredicate(5)));
+        assertParseSuccess(parser, " 7 ", new ShowNextCommand(new UpcomingEventPredicate(7)));
+        assertParseSuccess(parser, " 33", new ShowNextCommand(new UpcomingEventPredicate(33)));
+        assertParseSuccess(parser, "500", new ShowNextCommand(new UpcomingEventPredicate(500)));
+    }
```
Comment (anthonynmh) [inline src/test/java/ezschedule/logic/parser/ShowNextCommandParserTest.java:38]: Would it be possible for there to be comments on the equivalence partitions? Similar to `parse_invalidArgs_throwsParseException()`.

**Review by anthonynmh — CHANGES_REQUESTED** [review]
Left a comment for review. Once that resolves, should be GTM.

File: src/test/java/ezschedule/logic/parser/ShowNextCommandParserTest.java (hunk 1 above)
Comment (lhy-hoyin) [reply src/test/java/ezschedule/logic/parser/ShowNextCommandParserTest.java:40]: The equivalence partition can be inferred from the test name i.e. arbitrary numbers which are valid arguments.
To better understand the difference between the different asserts in here, I have added some comments of their differences.

**Comment by anthonynmh** [comment]
LGTM

**Pull request merged**